pkg/pillar/kubeapi/vitoapiserver.go

```diff
@@ -313,6 +313,92 @@ func RolloutDiskToPVC(ctx context.Context, log *base.LogObject, exists bool,
 	return nil
 }
 
+// GetPVFromPVC : Returns volume name (PV) from the PVC name
+func GetPVFromPVC(pvcName string, log *base.LogObject) (string, error) {
+	// Get the Kubernetes clientset
+	clientset, err := GetClientSet()
+	if err != nil {
+		err = fmt.Errorf("failed to get clientset: %v", err)
+		log.Error(err)
+		return "", err
+	}
+	// Get the PersistentVolumeClaim (PVC)
+	pvc, err := clientset.CoreV1().PersistentVolumeClaims(EVEKubeNameSpace).Get(context.TODO(), pvcName, metav1.GetOptions{})
```

Comment:
I'm just curious what is the `context.TODO()` in this case and will it be changed later...

Reply:
Also, I'm a little bit worried about the amount of dereferencing in one line. Could be hard to debug if one of them fails.

Reply:
The err should contain the actual error if something fails.